code/__HELPERS/radiation.dm

```diff
@@ -17,9 +17,9 @@
 /// period of time.
 /proc/radiation_pulse(
 	atom/source,
-	max_range,
+	max_range = null,
```

Comment:
Allows people to call radiation pulse without max_range defined in the parameters. Maybe I could not allow that since you aren't really a fan of people just using the intensity.

Reply:
`max_range` and `max_range = null` mean the exact same thing.